ionic-app/components/Menu.jsx

```diff
@@ -0,0 +1,80 @@
+import { StatusBar, Style } from '@capacitor/status-bar';
+import {
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonMenu,
+  IonMenuToggle,
+  IonTitle,
+  IonToolbar,
+} from '@ionic/react';
+import { useEffect, useState } from 'react';
+import { cog, flash, list } from 'ionicons/icons';
+
+const pages = [
+  {
+    title: 'Feed',
+    icon: flash,
+    url: '/tabs/feed',
+  },
+  {
+    title: 'Lists',
+    icon: list,
+    url: '/tabs/lists',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
    url: "/tabs/lists",
```